engine/engine/src/test/wscript

```diff
@@ -57,7 +57,7 @@ def build(bld):
 
     resource_type_symbols = ['ResourceTypeGameObject', 'ResourceTypeCollection', 'ResourceTypeScript', 'ResourceTypeLua',
                              'ResourceTypeAnim', 'ResourceTypeAnimationSet', 'ResourceTypeGui', 'ResourceTypeGuiScript']
-    component_type_symbols = ['ComponentTypeAnim', 'ComponentTypeScript', 'ComponentTypeGui']
+    component_type_symbols = ['ComponentTypeAnim', 'ComponentTypeScript', 'ComponentTypeGui', 'ComponentTypeMesh']
```

Comment:
The "build_input.yml" file is missing in this PR?

Reply:
We are talking about the C++ symbols, in your case `ComponentTypeMesh`.
It needs to be specified for the engine to be able to link with it.

See "symbols" at the top of the file, search for `ComponentTypeGui` as an example.

Reply:
Ok. I've added it. 

Note, it built and ran a test project with mesh properly even without this change.

Reply:
"Without this change"
Are you talking about build_input.yml?
This config file is used when you build a native extension (you have to use your sha1 version of the sdk)
Did you build a project with native extensions? With your sha1?